src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/config.groovy

```diff
@@ -6,10 +6,10 @@ f.entry(title: _("bitbucket.admin.token"), field: "secret") {
     f.password(value: instance?.secret, placeholder: "This token must have project admin permissions")
 }
 
-f.entry(title: _("bitbucket.admin.token.description"), field: "description") {
-    f.textbox(placeholder: "To help administrators identify this token")
+f.entry(title: _("bitbucket.admin.token.id"), field: "id") {
+    f.textbox(value: instance?.id,placeholder: "Enter a unique ID for this token")
```

Comment:
Maybe "Enter a unique ID or leave blank to generate"- we generate a UUID if the field is empty.

Reply:
/nit space between the comma and `placeholder`